docker-compose-with-zipkin.yaml

```
@@ -0,0 +1,29 @@
+version: '3.1'
+
+services:
+  db:
+    image: mysql:8.0.37-debian
```

I don't think it needs to be this specific:

what about either:
```suggestion
    image: mysql:lts
```

or:
```suggestion
    image: mysql:8.0-debian
```

---

I think I'll just remove that file. It doesn't really add anything.